RIS/astm-uam-psu-ris-0.0.0.yaml

```diff
@@ -0,0 +1,864 @@
+          enum: 
+          - DURATION
+          - LIMIT
+          - RATE
+        capacity_value:
+          type: number
+          description: |-
+            The number of flights allowed without exceeding the capacity limit.  This
+            number represents the maximum capacity for the limit type. 
+          minimum: 0
+    ResourceCapacity:
+      required:
+      - capacity_limit_type
+      - capacity_property_array
+      properties:
+        capacity_limit_type:
+          $ref: "#/components/schemas/CapacityLimitType"
+        capacity_property_array:
+          minItems: 1
+          maxItems: 2
+          type: array
+          description: |-
+            Describes the properties of the capacity requirements defined for the resource.
+          $ref: "#/components/schemas/CapacityPropertyType"
+    TemporaryResourceCapacity:
+      required: 
+        - id
+        - resource_capacity
+        - capacity_start_time
+      description: |-
+        A temporary resource capacity which overrides the default Resource Capacity. 
+      properties:
+        id:
+          description: |- 
+            A unique identifier for this temporary change to the default resource capacity.  Included
+            to allow for future updates (ex. adding an end time) to an existing temporary capacity change. 
+          $ref: '#../../PSU/astm-uam-psu-dss-0.0.0.yaml/components/schemas/EntityID'
+        resource_capacity:
+          $ref: "#/components/schemas/ResourceCapacity"
+        capacity_start_time:
+          description: The time at which the updated resource capacity begins. 
+          anyOf:
+          - $ref: '#../../PSU/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Time'
+        capacity_end_time:
+          description: The time at which the updated resource capacity ends, if known. 
+            If not provided, the temporary capacity continues indefinitely. 
+          anyOf:
+          - $ref: '#../../PSU/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Time'
+    ActiveResourceCapacity:
+      required: 
+        - resource_capacity
+      description: |-
+        The current resource capacity requirement in place. This may be a default or a 
+        temporary capacity requirement. 
+      properties:
+        resource_capacity:
+          $ref: "#/components/schemas/ResourceCapacity"
+        capacity_end_time:
+          description: The time at which the updated resource capacity ends, if known. 
+            If not provided, the current capacity continues indefinitely. 
+          anyOf:
+          - $ref: '#../../PSU/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Time'
+    FutureCapacities:
+      required:
+        - future_capacities
+      description: |-
+        List of upcoming capacity changes for the resource.  Should be ordered by time and progress
+        from earliest to latest. 
+      type: array
+      items:
+        $ref: "#/components/schemas/TemporaryResourceCapacity"
+    CurrentResourceStatus:
```

Comment:
I noticed that there's no schema or properties defined for future resource status (resource availability). Was this intentional, or is it something that still needs to be defined?